portfolio/src/main/webapp/script.js

```diff
@@ -42,4 +42,28 @@ async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
   document.getElementById('quote-container').innerText = quote;
+}
+
+async function loadTasks() {
+    let list = await fetch('/list-tasks').then(response => response.json());
+    for(c in list){ 
+        let reviewsElem = document.getElementById("reviews");
+        let commentElem = document.createElement("DIV");
+        commentElem.appendChild(document.createTextNode(list[c].email+": "+list[c].description));
+    }
+    document.getElementById("showCQC").onclick = null;
+}
+async function loginStatus() {
+    let status = await fetch('/status').then(response => response.json());
+    console.log(status);
+    if (status=='Currently logged in as Guest.'){
```

Comment:
You might also want to look into using an error code as opposed to parsing the string result to determine whether a user is logged in. This will help standardize how the backend communicates with the frontend and will be easier to keep the frontend/backend in sync.